internal/helm/controller/reconcile.go

```diff
@@ -388,9 +389,33 @@ func (r HelmOperatorReconciler) Reconcile(ctx context.Context, request reconcile
 		Manifest: expectedRelease.Manifest,
 	}
 	err = r.updateResourceStatus(ctx, o, status)
+
+	// Parse the reconcile period from CR's annotations
+	annotationsReconcilePeriod := parseReconcilePeriod(o)
```

Comment:
For the record, `o` is a horrible variable name for this but since it is an existing structure, I won't make you rewrite it here.